graylog2-web-interface/src/views/components/visualizations/GenericPlot.jsx

```diff
@@ -124,24 +124,28 @@ class GenericPlot extends React.Component<Props, State> {
       autosize: true,
       showlegend: true,
       margin: {
-        autoexpand: true,
         t: 10,
         l: 40,
         r: 10,
-        b: 40,
+        b: 0,
```

Comment:
I removed the margin, to provide more space for the chart. The widget padding already displays enough "margin".